src/frontend/components/SøknadsSteg/Dokumentasjon/Dokumentasjon.tsx

```diff
@@ -66,23 +74,27 @@ const Dokumentasjon: React.FC = () => {
     });
 
     const stegTekster = tekster()[ESanitySteg.DOKUMENTASJON];
-    const { dokumentasjonGuide } = stegTekster;
 
     return (
         <Steg
-            tittel={<SpråkTekst id={'dokumentasjon.sidetittel'} />}
-            guide={dokumentasjonGuide}
+            tittel={<TekstBlock block={stegTekster.dokumentasjonTittel} />}
+            guide={
+                brukerHarVedleggskrav
```

Comment:
Dette er logikk som også med fordel kan trekkes ut i en `dokumentasjon.domene.ts` fil

Reply:
Jeg prøvde å legge den til i den eksisterende utils filen for dokumentasjon, men fikk dependency cycles når jeg prøvde å kalle "useApp". Jeg la dokumentasjonGuide derfor inn i AppContext.

Reply:
Å legge til uthenting av tekster i AppContext fungerer heller ikke ettersom oppsett av Sanity skjer senere i bygge-prosessen. Jeg lurer på om håndtering av hvilken guide-tekst som skal brukes bør ligge direkte inne i Dokumentasjon.tsx med mindre det er en annen løsning.